### ISSUE TITLE
False positive: JSON encoding API usage

### ISSUE THREAD

**Issue by Bilge** [kalessil/phpinspectionsea#1873]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.7.1 |
| Language level | PHP 8.1                        |

```php
// OK
\json_decode($json, true, 512, JSON_THROW_ON_ERROR);

// [EA] Please consider taking advantage of JSON_THROW_ON_ERROR flag for this call options. 
\json_decode($json, true, flags: JSON_THROW_ON_ERROR);
```

**Comment by Bilge** [comment]
LOL

**Comment by jdreesen** [comment]
Yes, it seems to be the most itching issue in this project :grin:

**Comment by ea-inspections-team** [comment]
Thank you for reporting @Bilge  ! Fixed in #1977.
